README.md

```diff
@@ -1 +1,41 @@
-# weather_app
\ No newline at end of file
+# weather_app
+
+![Screenshot](./asset/Screenshot1.png)
+
+![Screenshot](./asset/Screenshot2.png)
```

Comment:
Screenshots are not showing

Reply:
https://user-images.githubusercontent.com/31889642/125740215-a6c0afc6-1541-4fa7-90d6-edf9f6f5b7cb.png

This error occurs because this file contains my API keys so in URL where the data is fetched you have to put your api keys

Reply:
Then kindly add the instructions in your readme file